client/src/pages/Home/Login.jsx

```diff
@@ -101,22 +101,25 @@ export default function Login() {
                 styles="w-full bg-primary-600"
                 func={handleSubmit}
               />
-              <Button
-                color="white"
-                text="Forgot Password"
-                borderRadius="10px"
-                width="full"
-                func={handleForgotPassword}
-              />
+              
               <p className="text-sm font-light text-gray-500 dark:text-gray-400">
+                Forgot Password?{" "}
+                <a
+                  href="/forgotPassword"
```

Comment:
Use ` href="#" 
onClick={handleForgotPassword}` // Call the function when clicked 
instead of `href="/forgotPassword"`